curator/defaults/client_defaults.py

```diff
@@ -24,6 +24,7 @@ def config_client():
         Optional('timeout', default=30): All(
             Coerce(int), Range(min=1, max=86400)),
         Optional('master_only', default=False): Boolean(),
+        Optional('api_key', default=False): Any(None, str, unicode),
```

Comment:
I suggest a default of `None` here. That's probably what's going on is that you have a Boolean value when you really want it to be empty.